code/game/gamemodes/shadowling/shadowling_abilities.dm

```diff
@@ -355,8 +370,8 @@
 			S.start()
 		qdel(B)
 
-/datum/reagent/shadowling_blindness_smoke //Blinds non-shadowlings, heals shadowlings/thralls
-	name = "!(%@ ERROR )!@$"
+datum/reagent/shadowling_blindness_smoke //Blinds non-shadowlings, heals shadowlings/thralls
+	name = "Odd Black Liquid"
```

Comment:
should be lower case, so BYOND will automatically append "the" at the front. alternatively, prefix this with \improper, although that has been known to be buggy.